app/controllers/pipeline_work_in_progress_controller.rb

```diff
@@ -0,0 +1,144 @@
+# frozen_string_literal: true
+
+class PipelineWorkInProgressController < ApplicationController
+  def index
+    # TODO: how can we avoid hardcoding this? Problem is there are multiple relevant pipelines we want to display in one.
```

Comment:
Not sure what "Problem is there are multiple relevant pipelines we want to display in one." means

Reply:
It means that ideally, I'd like to just give them a selection of pipelines to choose from, and display the plates for whichever one they choose (for in the future, when it's not limited to Heron).

There's a problem with that though, since for 'Heron' I want the page to display 'Heron-384 Tailed A' and 'Heron-384 Tailed B' - so, for one pipeline there might be multiple pipeline configs, and there's nothing to help the code figure out those groupings. I think it's just a case of adding something into the pipeline config like 'pipeline_group' or something, which this code could read. 

Not a priority for now though, I think hardcoding Heron is OK for first version. I'll make the comment clearer.

Reply:
Adding some config sounds like it would be the best way to do it.